recipes/tclap/1.2.3/conanfile.py

```diff
@@ -0,0 +1,19 @@
+from conans import ConanFile, tools
+
+class TclapConan(ConanFile):
+    name = "tclap"
+    version = "1.2.3"
+    license = "MIT License"
```

Comment:
Please you the SPDX short identifiers

```suggestion
    license = "MIT"
```